frontend/src/admin/components/editable-table/editable-table.tsx

```diff
@@ -0,0 +1,189 @@
+import {useState} from 'react'
+import EditRowIcon from '../edit-row-icon/edit-row-icon'
+import DeleteRowIcon from '../delete-row-icon/delete-row-icon'
+import SaveEditIcon from '../save-edit-icon/save-edit-icon'
+import CancelEditIcon from '../cancel-edit-icon/cancel-edit-icon'
+import {AccessRight} from '../../models/access-right'
+import {AccessRightGroup} from '../../models/access-right-group'
+import './editable-table.scss'
+
+export default function EditableTable({ accessRightGroups }: { accessRightGroups: AccessRightGroup[] }) {
+  const [editMode, setEditMode] = useState(false)
```

Comment:
I see. This is properly being changed in the next PR. Here the edit mode is being toggled via a button instead of toggling the edit mode on each row.